jenkins/ci.opensuse.org/openstack-submit.yaml

```
@@ -50,7 +50,7 @@
             ;;
           esac
 
-          if [ "Cloud:OpenStack:Rocky" = "$COS" ]; then
+          if [[ "$COS" =~ "(Rocky|Stein)" ]; then
```

```suggestion
          if [[ "$COS" =~ (Rocky|Stein) ]; then
```
Otherwise it won't match.